### PR TITLE
What it Takes to be a Flutter Developer.

### LINKED ISSUE #830: What it Takes to be a Flutter Developer.
### **Brief Summary:**
In a time and date in which technology allows for the creation of cross-platform applications despite a wide range of Operating systems available , it is important to shade light into some of these careers and have a look at who is responsible for creating such applications and how they do it. The article will looking into Flutter Developers as a profession and what their contribution to technology is. A deep dive into Flutter Developers and their use of Flutter open Source platform will be explained bringing to an understanding of what exactly is entailed in the profession.

### **Key Takeaways:** 

1. The article will bring a vast understanding of who a Flutter Developer is to the reader. 

2. The prior knowledge and skills needed to become a Flutter Developer will also be discussed in the article. 

3. It will explain the key roles and responsibilities of a Flutter Developer and what to expect on to do a daily basis pursuing the career. 

4.The article will describe in depth the tools used by a Flutter developer bring the reader to an understanding of the tools they should familiarize themselves with.

### **References:**
N/A.

### PR DESCRIPTION
**Pull request by bluemutheu**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

This fixes #830

### CODE REVIEW

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

**Comment by WanjaMIKE** [comment]
@hey I will reviewing this flutter article.  Your recent commits do not belong to this branch. Right?

**Comment by bluemutheu** [comment]
@WanjaMIKE yes, I have reverted them, had pushed using the same branch. The article is ready.

File: articles/becoming-a-flutter-developer/index.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+   - [Flutter Developer Interview Questions](#flutter-developer-interview-questions)
+- [Further Reading](#further-reading)
+- [Conclusion ](#conclusion)
+
+
+### Who is a Flutter Developer
+
+A Flutter developer is a software developer who develops cross-platform applications. They use `Flutter Framework` to develop these applications using a single base code. 
+
+### Roles and Responsibilites of a Flutter Developer
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:27]: Responsibilities

File: articles/becoming-a-flutter-developer/index.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
+`Flutter` is an open-source UI software development kit that `Google` released in 2018. It uses Dart as its primary programming language. Flutter has led to the emergence of new software developers called `Flutter Developers`. 
+
+### Introduction
+
+Cross-platform applications enable companies to cut down on development costs. It eliminates the need to hire different software development teams. They also reduce the time and cost to manage different code bases by using a single code base to develop applications.  `Flutter developers` develop these cross-platform applications.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:5]: It eliminates the need ... --- are we still discussing cross-platform applications? If so, `it` is wrong.

Comment (WanjaMIKE) [reply articles/becoming-a-flutter-developer/index.md:5]: Could you rephrase 
`Flutter developers` develop these cross-platform applications.
To
Flutter allows developers to build cross-platform applications

File: articles/becoming-a-flutter-developer/index.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+- [Getting a job as a Flutter Developer](#getting-a-job-as-a-flutter-developer)
+   - [Job Description Example](#job-description-example)
+   - [Flutter Developer Interview Questions](#flutter-developer-interview-questions)
+- [Further Reading](#further-reading)
+- [Conclusion ](#conclusion)
+
+
+### Who is a Flutter Developer
+
+A Flutter developer is a software developer who develops cross-platform applications. They use `Flutter Framework` to develop these applications using a single base code.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:25]: `A Flutter developer is a software developer who develops cross-platform applications` Avoid repetition of sentences -- same as line 5

File: articles/becoming-a-flutter-developer/index.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+### Roles and Responsibilites of a Flutter Developer
+
+1. **To Code Applications.**
+
+It is the process of writing the code to develop software. Flutter Developers write this code on the Flutter framework.
+
+2. **TO Test Applications.**
+
+They test the functionality of the applications to make sure it does not crash. It also ensures the apps are running the well based on the system and user requirements.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:35]: `It also ensures the apps are running the well based on the system and user requirements.` We are talking about flutter developers right? If so `it` is not right.

File: articles/becoming-a-flutter-developer/index.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+### Who is a Flutter Developer
+
+A Flutter developer is a software developer who develops cross-platform applications. They use `Flutter Framework` to develop these applications using a single base code. 
+
+### Roles and Responsibilites of a Flutter Developer
+
+1. **To Code Applications.**
+
+It is the process of writing the code to develop software. Flutter Developers write this code on the Flutter framework.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:31]: Could you please rephrase `Flutter Developers write this code on the Flutter framework.`  The Flutter Framework is more of a development kit which allows you to use different widgets. A better sentence will be something like
`Flutter developers use the Flutter SDK to craft beautiful and interactive applications`

File: articles/becoming-a-flutter-developer/index.md [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+
+
+### Flutter Developer Tools
+
+1. **Version Control Tools**
+
+These tools allow software developers to keep track of the changes they make on their projects. They also help for easy collaboration of a team of software developers while contributing to the same project. 
+
+Examples of version control tools:
+- [Git and Github](https://github.com/)
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:49]: Could you explain more on how a Flutter developer can use git and github

File: articles/becoming-a-flutter-developer/index.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+- [Bitbucket](https://bitbucket.org/)
+
+2. **Flutter Framework**
+
+Flutter is a UI framework used to develop high performance and high-quality native applications for Android, iOs, the Web and desktop using a single code base. Dart is the primary programming language for Flutter Framework. It is a free and open-source framework. 
+
+Some features of Flutter that make it popular include 
+
+1. `Hot Reloading` which enables developers to see changes from the codebase in the UI. 
+2. `Hot Restart`.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:59]: Please explain more on hot restart

File: articles/becoming-a-flutter-developer/index.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+Flutter is a UI framework used to develop high performance and high-quality native applications for Android, iOs, the Web and desktop using a single code base. Dart is the primary programming language for Flutter Framework. It is a free and open-source framework. 
+
+Some features of Flutter that make it popular include 
+
+1. `Hot Reloading` which enables developers to see changes from the codebase in the UI. 
+2. `Hot Restart`. 
+3. `Present UI element` to create beautiful app designs for better `UI and UX`. 
+
+These UI elements are
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:62]: Change `These UI elements are ...` to `These UI designs are....`  UI elements are mainly things such as Cards, buttons, etc. Also, explain more about material and Cupertino designs since they are part of Flutter's foundation

File: articles/becoming-a-flutter-developer/index.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+
+
+3. **Integrated Development Environment**
+
+In using Flutter, you will choose between three IDEs. Which are:
+- Android Studio.
+- Visual Studio Code.
+- IntelliJ IDEA.
+
+Most Flutter developers chose between Android Studio and Visual Studio Code. They chose IDEs depending on the features suitable to them. Android Studio and Visual Studio Code are both capable of creating full Flutter apps, developer preferences aside.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:76]: Could you use a synonym for chose to avoid repetition.

File: articles/becoming-a-flutter-developer/index.md [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+
+4. **DevTools**
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:78]: Could you provide specific examples of dev tools for the article to be more informative

File: articles/becoming-a-flutter-developer/index.md [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+
+It is a tooling web suite for Flutter and Dart developers comprising a layout inspection tools, performance tools and memory tools. It contains all the debugging tools that you need to be an efficient and effective Flutter developer. 
+
+5. **Design Tools**
+
+These are tools used to craft and visualize ideas. Design is an important stage of app development. It ensures for a smooth software development process. Some software development teams include UI/UX designers, especially for building commercial applications while others lack. Building personal projects will also need knowledge of design tools.
+
+Here are some of the most popular design tools:
+- Sketch.
+- Adobe XD.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:88]: Add links for Sketch and Adobe XD as well.

File: articles/becoming-a-flutter-developer/index.md [hunk 12]
```diff
@@ -0,0 +1,179 @@
...
+- [Figma.](https://www.figma.com/) 
+- [Flutter Studio.](https://flutterstudio.app/)
+
+After designing, you can use [Supernova](https://www.supernova.io/) to convert the design into code by exporting it. You will, however, need to refactor the exported code to break it into readable code.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:92]: Does supernova support all the listed designs (Figma, Adobe XD etc. Please clarify for the reader. You can also note that Sketch only works on macs.

File: articles/becoming-a-flutter-developer/index.md [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+
+### Flutter Developer Skills
+
+1. **Dart Programming Language**
+
+`Dart` is a client-based programming language that is scalable and can write simple scripts or full-featured applications. It is an object-oriented and dynamic language. Google developed dart. It is the primary language used in Flutter.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:98]: Could you change ```Dart` is a client-based programming language that is scalable and can write simple scripts `` to ``Dart` is a client-based programming language that is scalable and can be used to write simple scripts ``

File: articles/becoming-a-flutter-developer/index.md [hunk 14]
```diff
@@ -0,0 +1,179 @@
...
+
+2. **Knowledge on Agile Methodology and Waterfall for Software Development**
+
+A software development method is dividing the software development process into different phases. It enhances product design and resource utilization ***for example*** money and time. 
+
+The most commonly used methodologies are `Waterfall Methodology` and `Agile Methodology`. A Flutter developer needs to familiarize themselves with the software methodologies to work with ease in a software development team.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:104]: Could you provide a brief explanation of `Waterfall Methodology` and `Agile Methodology`

File: articles/becoming-a-flutter-developer/index.md [hunk 15]
```diff
@@ -0,0 +1,179 @@
...
+
+3. **Prior knowledge on a Native Language**
+
+The native languages for mobile application development include:
+
+1. `Swift` for iOS applications.
+
+2. `Java` and `Kotlin` for Android applications.
+
+Understanding these languages will make it easier to become a flutter developer. Learning Dart with the knowledge of a native language makes it easier.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:114]: Remove one sentence to avoid repetion.
`Understanding these languages will make it easier to become a flutter developer. Learning Dart with the knowledge of a native language makes it easier.`
Can simply go as
`Understanding these languages will make it easier to become a flutter developer.`

File: articles/becoming-a-flutter-developer/index.md [hunk 16]
```diff
@@ -0,0 +1,179 @@
...
+
+
+### Mistakes to Avoid When Learning Flutter Development
+
+- **Learn native languages, not just flutter and dart.**
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:119]: Ensure case is consistent. Its `Flutter` and not `flutter`.

File: articles/becoming-a-flutter-developer/index.md [hunk 17]
```diff
@@ -0,0 +1,179 @@
...
+
+Most software developers will argue that prior knowledge in a native language is unnecessary, but it is a skill requirement on most job descriptions. Flutter development is still fairly new, and employers must have proof that you know about application development. The only way to show this is by having experience with native languages were the only ones used to develop most commercial and large-scale applications.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:121]: Check this sentence.
`The only way to show this is by having experience with native languages were the only ones used to develop most commercial and large-scale applications.`
 It seems you were explaining one point and quickly jumped to another.

File: articles/becoming-a-flutter-developer/index.md [hunk 18]
```diff
@@ -0,0 +1,179 @@
...
+
+Knowing a native language will give you a groundbreaking compared to a person who only knows flutter and dart. It will display a variety of expertise and knowledge. A grasp on native languages will make it easier to move flutter development.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:123]: In this sentence
`Knowing a native language will give you a groundbreaking compared to a person who only knows flutter and dart`
`groundbreaking` can simply be replaced with `advantage`. Some users may not be familiar with that term.

File: articles/becoming-a-flutter-developer/index.md [hunk 19]
```diff
@@ -0,0 +1,179 @@
...
+
+- **Do not over-rely on one source to develop your career as a Flutter Developer.**
+
+Flutter and Dart are still fairly new and not fully adapted. Finding a tutorial or reading material with a sufficient amount of knowledge to kick off your career might prove to be a bit hard. It is advisable to look into multiple resources to get a good grasp of the required knowledge. Different tutors have unique experiences and knowledge of the framework and programming language.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:127]: I believe Flutter is fully adapted now. They released their `beta` releases some years ago. Now, they release `stable` versions or updates regularly. Check out companies using it https://flutter.dev/showcase

File: articles/becoming-a-flutter-developer/index.md [hunk 20]
```diff
@@ -0,0 +1,179 @@
...
+
+- **Do not learn flutter in general, pick an app to develop from scratch upwards.**
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:129]: flutter - case error

File: articles/becoming-a-flutter-developer/index.md [hunk 21]
```diff
@@ -0,0 +1,179 @@
...
+
+Avoid confusion when trying to get into a new profession. Do not learn the framework generally instead find an example of an application you would like to build and build it from scratch upwards. It will allow you to make mistakes, learn from them, and have an actual project to add into your portfolio. Learning through building an actual application will prepare you for a real-world experience into the profession.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:131]: Check out this sentence
`Do not learn the framework generally instead find an example of an application you would like to build and build it from scratch upwards. It will allow you to make mistakes`
The `build and build` part reduces its quality

File: articles/becoming-a-flutter-developer/index.md (hunk 9 above)
Comment (bluemutheu) [reply articles/becoming-a-flutter-developer/index.md:76]: @WanjaMIKE synonym for which word exactly?

File: articles/becoming-a-flutter-developer/index.md [hunk 22]
```diff
@@ -0,0 +1,179 @@
...
+- To start on [Dart programming language](https://www.udemy.com/course/dart-beginners-course/?couponCode=7CA93B47A390A81961E3).
+
+- To start on [Flutter Framework](https://flutter.dev/).
+
+- To learn more about `DevTools`
+
+<iframe width="478" height="269" src="https://www.youtube.com/embed/nq43mP7hjAE" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+### Conclusions
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:175]: `Conclusion` rather than `Conclusions`

File: articles/becoming-a-flutter-developer/index.md [hunk 23]
```diff
@@ -0,0 +1,179 @@
...
+To become a flutter developer is simpler than becoming other professions in the software engineering field. If one knows development using native languages, it will be easier. Dart is a friendly language and Flutter an easy-to-understand framework. Becoming a Flutter Developer is a very interesting, fun and the field is still growing as the industry gets more comfortable with Flutter framework.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:176]: Can you say `becoming a Flutter developer is simple`. instead of comparing it with other professions. I am sure some readers will disagree with the original sentence.

**Review by WanjaMIKE — CHANGES_REQUESTED** [review]
Hey @bluemutheu 
The article has some grammar issues that need to be resolved. Could you please check out the suggestions or comments. Thanks

File: articles/becoming-a-flutter-developer/index.md (hunk 19 above)
Comment (bluemutheu) [reply articles/becoming-a-flutter-developer/index.md:127]: @WanjaMIKE I did not mean that it is not being used by companies, of course it is. I meant the knowledge and level of understanding of the framework is not sufficient to make a learner rely on one source, most questions even on stack overflow are unanswered, its about 3-4 years old, with a slow adaption rate. or what did you mean?

File: articles/becoming-a-flutter-developer/index.md (hunk 9 above)
Comment (WanjaMIKE) [reply articles/becoming-a-flutter-developer/index.md:76]: `chose`

File: articles/becoming-a-flutter-developer/index.md (hunk 19 above)
Comment (WanjaMIKE) [reply articles/becoming-a-flutter-developer/index.md:127]: Okay. You can rephrase the sentence then. Thanks

**Comment by bluemutheu** [comment]
@WanjaMIKE  Thank you for the suggestions. I have made the changes.

File: articles/becoming-a-flutter-developer/index.md [hunk 24]
```diff
@@ -0,0 +1,188 @@
...
+
+These are tools used to craft and visualize ideas. Design is an important stage of app development. It ensures for a smooth software development process. Some software development teams include UI/UX designers, especially for building commercial applications while others lack. Building personal projects will also need knowledge of design tools.
+
+Here are some of the most popular design tools:
+- [Sketch.](https://www.sketch.com/) This is however only available for Mac.
+- [Adobe XD.](https://www.sketch.com/)
+- [Figma.](https://www.figma.com/) 
+- [Flutter Studio.](https://flutterstudio.app/)
+
+After designing, you can use [Supernova](https://www.supernova.io/) to convert the design into code by exporting it. You will, however, need to refactor the exported code to break it into readable code. Supernova is compatible with the design tools mentioned above.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:101]: You can remove this part entirely.
`After designing, you can use [Supernova](https://www.supernova.io/) to convert the design into code by exporting it. You will, however, need to refactor the exported code to break it into readable code. Supernova is compatible with the design tools mentioned above. ` 
It is not much involved in `what it takes to be a Flutter developer`.

File: articles/becoming-a-flutter-developer/index.md [hunk 25]
```diff
@@ -0,0 +1,188 @@
...
+
+The native languages for mobile application development include:
+
+1. `Swift` for iOS applications.
+
+2. `Java` and `Kotlin` for Android applications.
+
+Understanding these languages will make it easier to become a flutter developer.
+
+### Mistakes to Avoid When Learning Flutter Development
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:128]: Could you adjust the sub sections to align with the section heading.
`### Mistakes to Avoid When Learning Flutter Development
- **Learn native languages, not just Flutter and Dart.**]
- **Learn native languages, not just Flutter and Dart.*`
It should be
`### Mistakes to Avoid When Learning Flutter Development
- **Ignoring native languages.**]

File: articles/becoming-a-flutter-developer/index.md [hunk 26]
```diff
@@ -0,0 +1,176 @@
...
+
+These UI designs are 
+1. Cupertino 
+2. Material Design. 
+
+These built-in features enhance the development of Flutter applications. 
+
+3. **Integrated Development Environment**
+
+The three popular IDEs that support Flutter are:
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:67]: Add links to these IDEs

File: articles/becoming-a-flutter-developer/index.md [hunk 27]
```diff
@@ -0,0 +1,176 @@
...
+2. Timeline - It diagnoses performance problems which are the most common kinds during development. The Timeline shows the amount of time taken by each widget.
+
+
+5. **Design Tools**
+
+These are tools used to craft and visualize ideas. Design is an important stage of app development. It supports a smooth software development process. Some software development teams include UI/UX designers, especially for building commercial applications while others lack. Building personal projects will also need knowledge of design tools.
+
+Here are some of the most popular design tools:
+- [Sketch.](https://www.sketch.com/) This is however only available for Mac.
+- [Adobe XD.](https://www.sketch.com/)
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:91]: The link for Adobe XD is wrong.

File: articles/becoming-a-flutter-developer/index.md [hunk 28]
```diff
@@ -0,0 +1,176 @@
...
+
+2. **Flutter Framework**
+
+Flutter is a UI framework used to develop high performance and high-quality native applications for Android, iOS, web and desktop using a single code base. Dart is the primary programming language for Flutter Framework. 
+
+Some features of Flutter that make it popular include 
+
+1. `Hot Reloading` which enables developers to load code changes quickley and preserves the state of the app. 
+2. `Hot Restart`. This is similar to hot reloading but causes the app to lose its state and takes more time.
+3. `Present UI element` to create beautiful app designs for better `UI and UX`.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:57]: Rephrase
``Present UI element` to create beautiful app designs for better `UI and UX`. `
To
`Availability of widgets which can be easily be imported into Flutter projects`

File: articles/becoming-a-flutter-developer/index.md [hunk 29]
```diff
@@ -0,0 +1,176 @@
...
+
+### Flutter Developer Tools
+
+1. **Version Control Tools**
+
+These tools allow software developers to keep track of changes they make on their projects. They also boost collaboration among software developers. As a result, they help increase overall productivity. 
+
+Examples of version control tools:
+
+[Git and Github](https://github.com/) - To use git and github you will need to install git to the code editor or IDE you are using and have an existing github account. All you need to do is write git commands to push your code to a github repository. From the repository other developers can help you improve your code. You can also set your repository as private.
```
Comment (WanjaMIKE) [inline articles/becoming-a-flutter-developer/index.md:45]: please rephrase this statement.
`To use git and github you will need to install git to the code editor or IDE you are using and have an existing github account.`
Do I need to install git to use github?

Comment (bluemutheu) [reply articles/becoming-a-flutter-developer/index.md:45]: @WanjaMIKE please explain how it's possible to use github without git on your local computer

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
Great article @bluemutheu  I think the community will find it informative. 

Please go through the minor changes I made for the format to apply to future articles. 

cc @WanjaMIKE  

Thank you 🚀

**Comment by WanjaMIKE** [comment]
@hectorkambow Okay, thank you
